ecs-cli/modules/cli/compose/project/project_parseV3.go

```diff
@@ -51,24 +75,90 @@ func (p *ecsProject) parseV3() (*[]containerconfig.ContainerConfig, error) {
 		return nil, err
 	}
 
-	// convert ServiceConfigs to ContainerConfigs
-	conConfigs := []containerconfig.ContainerConfig{}
-	for _, service := range config.Services {
-		cCon, err := convertToContainerConfig(service)
+	return config, nil
+}
+
+func convertToContainerConfig(serviceConfig types.ServiceConfig) (*containerconfig.ContainerConfig, error) {
+	//TODO: Add Healthcheck, Devices to ContainerConfig
+	c := &containerconfig.ContainerConfig{
```

Comment:
Total nit: could we possibly keep these alphabetized/in the same order as they're defined in the struct in`containerconfig.go`?

Reply:
talked offline: will alphabetize within the initialization block, then among individually set fields